platform/security/handler/security-handler-pki/src/test/java/org/codice/ddf/security/handler/pki/PKIHandlerTest.java

```diff
@@ -46,57 +46,15 @@
     @Test
     public void testGetNormalizedTokenSuccessNoCrlPki()
             throws java.security.cert.CertificateException, ServletException {
-        for (AbstractPKIHandler handler : new AbstractPKIHandler[] {new PKIHandler(),
-                new WssPKIHandler()}) {
-            PKIAuthenticationTokenFactory tokenFactory = new PKIAuthenticationTokenFactory();
-            tokenFactory.setSignaturePropertiesPath("signatures.properties");
-            tokenFactory.init();
-            handler.setTokenFactory(tokenFactory);
-
-            HttpServletRequest request = mock(HttpServletRequest.class);
-            HttpServletResponse response = mock(HttpServletResponse.class);
-            FilterChain chain = mock(FilterChain.class);
-
-            String certificateString = getLocalhostCert();
-
-            InputStream stream = new ByteArrayInputStream(
-                    Base64.decodeBase64(certificateString.getBytes()));
-            CertificateFactory factory = CertificateFactory.getInstance("X.509");
-            X509Certificate cert = (X509Certificate) factory.generateCertificate(stream);
-            X509Certificate[] certs = new X509Certificate[1];
-            certs[0] = cert;
-            when(request.getAttribute(("javax.servlet.request.X509Certificate"))).thenReturn(certs);
-
-            /**
-             * Note that the getNormalizedToken() method for PKI handlers do not
-             * use the resolve tag.
-             */
-            HandlerResult result = null;
-            result = handler.getNormalizedToken(request, response, chain, true);
-
-            assertThat(result, is(notNullValue()));
-            assertThat(result.getStatus(), equalTo(HandlerResult.Status.COMPLETED));
-        }
-    }
-
-    /**
-     * This test ensures the proper functionality of PKIHandler's method,
-     * getNormalizedToken(), when given a valid HTTPServletRequest with
-     * invalid data.
-     */
-    @Test
-    public void testGetNormalizedTokenFailureNoCertBytes()
-            throws java.security.cert.CertificateException, ServletException {
-        PKIHandler handler = new PKIHandler();
-        PKIAuthenticationTokenFactory tokenFactory = new PKIAuthenticationTokenFactory();
-        tokenFactory.setSignaturePropertiesPath("signatures.properties");
-        tokenFactory.init();
-        handler.setTokenFactory(tokenFactory);
+        PKIHandler handler = getPKIHandlerWithMockedCrl("signature.properties", true);
```

Comment:
We should have `verify(crlChecker).passesCrl(`expected certs`)` checks at the end of each test to make sure `CrlChecker` is called or not called. To verify it didn't get called, you can use `verify(crlChecker, never()).passesCrl()`.